docs/general/dql/selects.rst

```diff
@@ -296,6 +296,20 @@ These operators can be used to query for rows where only part of a columns
 value should match something. The only difference is that, in the case of
 ``ILIKE``, the matching is case insensitive.
 
+Both ``LIKE`` and ``ILIKE`` support optional ``ESCAPE`` character. When no
+value or empty value is provided, backslash character ``\`` is used as escape
```

Comment:
Psql states the following for empty value:
```
It's also possible to select no escape character by writing ESCAPE ''. This effectively disables the escape mechanism, which makes it impossible to turn off the special meaning of underscore and percent signs in the pattern.
According to the SQL standard, omitting ESCAPE means there is no escape character (rather than defaulting to a backslash), and a zero-length ESCAPE value is disallowed. PostgreSQL's behavior in this regard is therefore slightly nonstandard.
```

Reply:
Good catch, thanks! I saw it but apparently misunderstood. I will push a change to disable escaping on empty ESCAPE (align with PG).

Reply:
Done in https://github.com/crate/crate/pull/15166/commits/b3ccfff071daf2e9acb9a0b6abe498baec233630